src/test/java/com/amazon/opendistroforelasticsearch/sql/antlr/semantic/SemanticAnalyzerScalarFunctionTest.java

```diff
@@ -95,6 +96,7 @@ public void logFunctionCallWithUnknownFieldShouldPass() {
         validate("SELECT LOG(new_field) FROM semantics");
     }
 
+    @Ignore /** Test set to ignore due to nested functions not supported and blocked by throwing SqlFeatureNotImplementedException */
```

Comment:
I think you can put the comment into `reason` field, ex.`@Ignore("Test set ...")`